lang/test/org/partiql/lang/eval/NaturalExprValueComparatorsTest.kt

```diff
@@ -64,6 +65,7 @@ class NaturalExprValueComparatorsTest : EvaluatorTestBase() {
             "-0.0",
             "`-0.0000000000`",
             "`0e0`",
+            "`-0e0`",
             "`0d10000`",
             "0"
```

Comment:
I would add `-0` too here.

Reply:
Added. `-0` is not an ion value though you can only have a float or decimal negative zero

Reply:
My reading of the section on Ion Int has -0 as an example 
http://amzn.github.io/ion-docs/docs/spec.html#int